pulp_2to3_migration/app/plugin/content.py

```diff
@@ -251,22 +226,16 @@ def get_remote_by_importer_id(importer_id):
             code='migrating.{}.content'.format(self.migrator.pulp2_plugin),
             total=pulp_2to3_detail_qs.count()
         ) as pb:
-            prefetch_args = [
-                Prefetch('pulp2content'),
-                Prefetch('pulp2content__pulp3_content'),
+            select_extra = [
+                'pulp2content',
+                'pulp2content__pulp3_content',
             ]
+
             if content_model.set_pulp2_repo:
-                prefetch_args.append(Prefetch('pulp2content__pulp2_repo'))
-            # Warning: It's dangerous to save records of the type of pulp_2to3_detail_qs
-            # while using this iterator due to the need for globally-accurate ordering.
-            # We're using the PK which is not an incrementing integer so that doesn't hold
-            # true. However, at this point, all records have already been saved so we are safe.
-            chunked_iterator = chunked_queryset_iterator(
-                pulp_2to3_detail_qs.prefetch_related(*prefetch_args),
-                2000
-            )
+                select_extra.append('pulp2content__pulp2_repo')
 
-            for pulp_2to3_detail_content in chunked_iterator:
+            pulp_2to3_detail_qs = pulp_2to3_detail_qs.select_related(*select_extra)
```

Comment:
I'm honestly not sure why I didn't write it like this originally.  Maybe I thought ```.select_related()``` also couldn't be used with ```.iterator()``` when it definitely can.